### PR TITLE
Porter#1141

### LINKED ISSUE #1141: Add support to the Kubernetes Mixin to support --context
Looking at the Kubernetes mixin to add the clientVersion similar to the Helm3 mixin and I came upon another missing construct I need the --context. Now I can add it at the Kubernetes level but am thinking it should be in the schema on the action level in case your trying to do something thats potentially multi-cluster.
I might have missed the existing option to do it though. If not

* it needs an issue (which I will add)
* thoughts on where it should live
* if it's on the root mixin would 1 PR for both be okay? If it's at the action level I'd definitely do 2 PRs

### PR DESCRIPTION
**Pull request by MChorfa**
# What does this change
Add `--context` arg to the schema 

# What issue does it fix
Closes #1141

[1]: /CONTRIBUTING.md#when-to-open-a-pull-request

# Notes for the reviewer
- These changes were made with `go version go1.14.4 darwin/amd64`. where I am having some unstable issues not related to changes made. Please wait until I confirm the environment behavior
# Checklist
- [x] Unit Tests
- [ ] Documentation
- [ ] Schema (porter.yaml)

If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️

[contributors]: /CONTRIBUTORS.md

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,6 +47,7 @@ build-mixin-%: generate
 	$(MAKE) $(MAKE_OPTS) build MIXIN=$* -f mixin.mk
 
 generate: packr2
+	$(GO) mod tidy
```
[the remark this answers is not in the document]
Comment (MChorfa) [reply Makefile:50]: The execution command is built prior with : `command, err := m.buildInstallCommand(args.InstallArguments, manifestPath)`. So if the context is passed into the install command, it gets passed to the execution as you can see from the struct https://github.com/deislabs/porter/blob/8db8b924f44ae5acd4143227ee4e9ddd393c5200/pkg/kubernetes/execute.go#L50

[the remark this answers is not in the document]
Comment (vdice) [reply Makefile:50]: Indeed, you're right.  Thank you for linking and pointing this out.

[the remark this answers is not in the document]
Comment (vdice) [reply Makefile:50]: Indeed, you're right.  Thank you for linking and pointing this out.  (And adding a unit test!)

[the remark this answers is not in the document]
Comment (MChorfa) [reply Makefile:50]: All good from my end

**Comment by vdice** [comment]
Thanks @MChorfa !

**Pull request merged**